woocommerce/woocommerce
# ISO Code update for Odisha state (India)

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

ISO Code change for Odisha state (India)



ISO Code update for Odisha state (India) https://en.wikipedia.org/wiki/ISO_3166-2:IN (23 November 2023)


### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Add products to the cart and proceed to checkout.
2. On the Country dropdown select `India` and on the State select `Odisha`.
3. Check the code for the select is `OD` instead of `OR`.

<img width="1016" alt="Screenshot 2024-12-10 at 09 50 08" src="https://github.com/user-attachments/assets/1fb8a654-4bdd-4efb-a5bd-b3b6d537f905">

### Changelog entry
Updated Odisha ISO Code - 23 November 2023












<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
ISO Code update for Odisha state (India) https://en.wikipedia.org/wiki/ISO_3166-2:IN (23 November 2023)
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
